deploy/examples/external/cluster-external.yaml

```
@@ -0,0 +1,32 @@
+#################################################################################################################
+# Define the settings for the rook-ceph-external cluster with common settings for a production cluster.
+#
+# For example, if Rook is not managing any existing cluster in the 'rook-ceph' namespace do:
+#   kubectl create -f ../../examples/crds.yaml -f ../../examples/common.yaml -f ../../examples/operator.yaml
+#   kubectl create -f common-external.yaml -f cluster-external.yaml
+#
+# If there is already a cluster managed by Rook in 'rook-ceph' then do:
+#   kubectl create -f common-external.yaml
+#   kubectl create -f common-external.yaml
+#################################################################################################################
+apiVersion: ceph.rook.io/v1
+kind: CephCluster
+metadata:
+  name: rook-ceph-external
```

are we not updating the namespace?
@travisn thoughts?

---

And we are not updating so we can use the symlink here as well, for some of the cases

---

According to this I believe we should make all the new docs with `rook-ceph` namespace https://github.com/rook/rook/issues/12599#issuecomment-1939378331

---

>then I need to keep the cluster name same as rook-ceph

Can you elaborate on this what do you mean by changing the cluster name?

---

@smoshiur1237 ^^

---

I want to confirm that if I need to change the namespace to `rook-ceph`, do I need to change the cluster name  same as namespace?

---

no need to change the cluster name

---

Do I change the namespace from` rook-ceph-external` to `rook-ceph` for all instance in all the files ?

---

@parth-gr  So you mean that I can change the namespace from `rook-ceph-external` to `rook-ceph` for all  manifest or other files under `deploy/examples ` folder including the new `external ` folder. So there will be no use of `rook-ceph-external` namespace .

---

No only make changes inside the ` new external  folder.` and don't touch other files

---

Agreed, I thought we decided to change the namespace in the `external` folder to `rook-ceph`, like this:
```suggestion
  name: rook-ceph
```